src/pages/auth/signin.tsx

```diff
@@ -1,3 +1,116 @@
-function SignInPage() {}
+import { useRouter } from 'next/router';
+import { useState } from 'react';
+import AuthInput from '@/components/input/authInput';
+import { useForm, SubmitHandler } from 'react-hook-form';
+import Image from 'next/image';
+import login from '@/services/Auth.API';
+
+type FormValues = {
+  email: string;
+  password: string;
+};
+
+/**
+ * 테스트 계정
+ * {
+  "email": "test1234@testuser.com",
+  "nickname": "test user",
+  "password": "Test1234!",
+  "passwordConfirmation": "Test1234!"
+}
+*/
+function SignInPage() {
+  const router = useRouter();
+  const { control, handleSubmit } = useForm<FormValues>();
+  const [error, setError] = useState<string | null>(null);
+
+  const onSubmit: SubmitHandler<FormValues> = async (data) => {
+    const { email, password } = data;
+
+    try {
+      // 로그인 API 요청
+      const response = await login({ email, password });
+      localStorage.setItem('accessToken', response.accessToken);
+      // 로그인 성공 시 팀 페이지로 리다이렉트 (팀 페이지 없어서 임시 페이지 설정)
+      router.push('/board/index.tsx');
+    } catch (err: unknown) {
+      // 에러 처리
+      if (err instanceof Error) {
+        setError(err.message || '이메일 또는 비밀번호가 잘못되었습니다.');
+      }
+    }
+  };
+
+  return (
+    <div className='flex justify-center items-center h-screen bg-transparent'>
+      <div className='w-full max-w-md bg-transparent p-8 rounded-md shadow-md'>
+        <h2 className='block text-40 text-text-primary text-center font-500 h-48 mb-80 leading-48'>
```

Comment:
폰트 크기 같은 경우에는 이미 설정되어있는 스타일을 사용해주세요~~!! 이렇게 설정하시면 line-height도 따로 설정해야 하니까요~~
그리고 h2가 블록 요소라서 반응형으로 처리하실 거 아니면 따로 block은 없어도 될거같아요~~!
```suggestion
        <h2 className='text-4xl text-text-primary text-center font-semibold h-48 mb-80'>
```